api/services/tagPresentation.js

```diff
@@ -0,0 +1,31 @@
+export const fetchAndPresentTagJoins = (joinClass, communityId, userId) =>
```

Comment:
the other presenters are also just collections of functions

On Thu, May 12, 2016 at 5:07 PM Robbie Carlton <notifications@github.com>
wrote:

> In api/services/tagPresentation.js
> <https://github.com/Hylozoic/hylo-node/pull/124#discussion_r63117044>:
>
> > @@ -0,0 +1,31 @@
> > +export const fetchAndPresentTagJoins = (joinClass, communityId, userId) =>
>
> the reason I didn't call it TagPresenter is because it's a collection of
> functions, not a class.
> You want me to put the functions in a class called TagPresenter, or just
> rename the file TagPresenter?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/124/files/eb7c945d8e7a485596dbae2ae335f3d0340efea8#r63117044>
>